src/main/target/STM32_UNIFIED/target.h

```diff
@@ -218,6 +241,40 @@
 // to be supplied when the board is configured for the first time
 #define USE_UNIFIED_TARGET
 
+// Only enable ELRS support on STM32F411SX1280 target
```

Comment:
what if someday a vendor wanna make a SX1280 SPI FC based on other chip, for example F745?

Reply:
If it has more than 512K of flash then SPI ELRS support will be included. If somebody makes an F722 FC with SPI ELRS then we’ll need to add an STMF7SX1280 target.